client/executor/wasmtime/src/instance_wrapper.rs

```diff
@@ -114,11 +112,7 @@ pub struct InstanceWrapper {
 	// proof See `memory_as_slice` and `memory_as_slice_mut`.
```

Comment:
Correct; there are multiple copies of `Memory` anyway (but it's fine since that's how it's supposed to be used; the memory is actually kept within the `Store`, and this is just the index of the slot within the `Store` where the memory is kept)